src/main/java/org/mifos/connector/channel/camel/routes/ChannelRouteBuilder.java

```diff
@@ -328,7 +328,7 @@ private void transferRoutes() {
                     extraVariables.put("clientCorrelationId", clientCorrelationId);
                     extraVariables.put("initiatorFspId", channelRequest.getPayer().getPartyIdInfo().getFspId());
                     String tenantSpecificBpmn;
-                    String bpmn = getWorkflowForTenant(tenantId);
+                    String bpmn = getWorkflowForTenant(tenantId, "payment-transfer");
                     if (channelRequest.getPayer().getPartyIdInfo().getPartyIdentifier().startsWith("6666")) {
```

Comment:
this was by default, a logic that even I am unsure about, since we do not have a logic that specifies a specific workflow based on party identifier value